app/controllers/receipt_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ReceiptController < ApplicationController
+  def index
+    @name = params[:name]
```

Comment:
I don't think a receipt needs a name.  When I get a receipt for something, it might have items listed, a total, tax, but not a name.